### ISSUE TITLE
Extended config file

### ISSUE THREAD

**Issue by jonathanmorley** [zuazo/ssl_certificate-cookbook#18]
For docker client auth, the certificates need to have the following attribute:

extendedKeyUsage = clientAuth

I don't see a way to add that to a certificate currently

**Comment by zuazo** [comment]
Thanks @jonathanmorley. Supporting X509 certificate extensions would be a good idea. Not currently supported.

For future reference, Docker documentation about this is here:

* [Protect the Docker daemon socket: Create a CA, server and client keys with OpenSSL](extendedKeyUsage).

**Comment by catalinvr** [comment]
+1

**Comment by zuazo** [comment]
Implemented in #28 and already merged onto master.

Usage example:

```ruby
ssl_certificate 'docker' do
  extended_key_usage %w(clientAuth)
end
```

**Comment by zuazo** [comment]
Released in [`2.0.0`](https://github.com/zuazo/ssl_certificate-cookbook/releases/tag/2.0.0)
